packages/amplication-server/src/core/core.module.ts

```diff
@@ -31,6 +31,7 @@ import { GptModule } from "./gpt/gpt.module";
 import { UsageInsightsModule } from "./UsageInsights/usageInsights.module";
 import { AssistantModule } from "./assistant/assistant.module";
 import { AwsMarketplaceModule } from "./aws-marketplace/aws-marketplace.module";
+import { PackagesModule } from "./packages/packages.module";
```

Comment:
Since all other blocks seem to be in singular form here (e.g. "ModuleDtoModule" and not "ModuleDtosModule", I think this should also be called "PackageModule"